R/plot.R

```diff
@@ -0,0 +1,323 @@
+#' @title Static rasterly graphics
+#' @description Render a \code{rasterly} object. 
+#' This object can involve axes, legend, etc to better convert graphics to data.
```

Comment:
Just a suggestion, assuming I understand the purpose of this function:

```suggestion
#' This function allows users to add axes, legends and other descriptive details when generating `rasterly` objects.
```

Reply:
Yes, thanks